app/src/test/kotlin/com/intive/picover/auth/intent/builder/SignInIntentBuilderTest.kt

```diff
@@ -21,13 +20,12 @@ class SignInIntentBuilderTest : ShouldSpec(
 		should("return intent wrapped by SignInIntent WHEN build called") {
 			val providers: AuthUI.IdpConfig = mockk()
 			val signInIntent: Intent = mockk()
-			val firebaseSignInIntentBuilder: FirebaseSignInIntentBuilder = mockk {
-				every { setTheme(R.style.Theme_Picover) } returns this
-				every { setAvailableProviders(listOf(providers)) } returns this
-				every { build() } returns signInIntent
-			}
 			val authUi: AuthUI = mockk {
-				every { createSignInIntentBuilder() } returns firebaseSignInIntentBuilder
+				every { createSignInIntentBuilder() } returns mockk {
+					every { setTheme(R.style.Theme_Picover) } returns this
```

Comment:
Personally I would drop passing exact theme here, it's not important from unit test perspective

Reply:
I think we can do it in the scope of this PR

Reply:
Done